CHANGELOG.md

```diff
@@ -5,8 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+
 ## [Unreleased]
 
+### Changed
+
+- Prepare project structure for `v1.0.0` by having all go code in `pkg/`.
```

Comment:
```suggestion
- Prepare project structure for `v1.0.0` by having all go code in `pkg/k8sclient/`.
```

I have the same remark as here https://github.com/giantswarm/k8scloudconfig/pull/692#discussion_r408005988